src/views/Main/Catalog/Components/PropertyList.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import { Property } from './Property';
+import '../styles.module.css';
+
+export class PropertyList extends React.Component {
+  static propTypes = {
+    properties: PropTypes.string,
+    primaryKey: PropTypes.string
+  }
+
+  keyProperties() {
+    const properties = JSON.parse(this.props.properties);
+    properties.map((prop) => {
```

Comment:
`.map()` returns an array of values that have been transformed by the given function, but you're not doing anything with it, and then you return properties.

I don't think this function does what you intended it to.